README.md

```diff
@@ -214,6 +215,22 @@ jobs:
       - uses: actions/checkout@v2
 ```
 
+## Checkout pull request on `pull_request_target`
+
+```yaml
+on:
+  - pull_request_target
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: refs/pull/${{ github.event.pull_request.number }}/head
```

Comment:
I would suggest this because it's more universal and works both with push and pull request triggers at the same time.
```suggestion
          repository: ${{ github.event.pull_request.head.repo.full_name }}
          ref: ${{ github.head_ref }}
```

Reply:
@matfax I intended it to be `merge` not `head`, which is more consistent with `pull_request` event. I'll update the PR. `github.head_ref` is only availabe for `pull_request` events.

Reply:
Both contexts are only available for pull requests. But the empty reference will automatically let the checkout action fallback to the master branch of the context repository. That's how it works with both. This won't work with 'refs/pull' though because it's not empty with a missing reference.